### ISSUE TITLE
[feat] Disable find in page

### ISSUE THREAD

**Issue by Jamzy01** [tauri-apps/tauri#7891]
### Describe the problem

Tauri has come a long way in building apps that almost seem native, but there is one glaring issue. Every time I press F3 or in the less likely case ctrl + f, the find in page feature appears. This immediately ruins the illusion and reminds me that it is just a website. It really is a small thing, but considering that opening the downloads page through ctrl + j is already disabled, I assume this could be a simple fix.

### Describe the solution you'd like

In the tauri.conf.json, it would be nice if you could disable inbuilt browser shortcuts individually. This could also help by allowing developers to disable ctrl + r, which personally doesn't bother me, but again just reminds you that it is just a web page.

### Alternatives considered

event.preventDefault() on these shortcuts does disable them, but this is in no way a clean and elegant solution.

### Additional context

As I explained in alternatives considered, it's not like this isn't already possible, so this feature isn't really in any way urgent. Still if it's a pretty simple feature to implement, then it would be nice to have in a future version of tauri.

**Comment by Smit1610** [comment]
I can work on this issue can you please assign to me ?

**Comment by Jamzy01** [comment]
> I can work on this issue can you please assign to me ?

Sorry, I don't think I can. I'm pretty sure only admins can assign issues.

**Comment by Tnze** [comment]
Also disable `Ctrl + R` please

**Comment by WindLeaf233** [comment]
Any solutions?

**Comment by Andrew-web-coder** [comment]
Any solution to this issue?

**Comment by Tnze** [comment]
> Any solution to this issue? 

Eat the keydown event in your javascript.
For example, to disable `F5`:

```typescript
document.addEventListener('keydown', event => {
    if (event.key === 'F5') {
        event.preventDefault()
    }
})
```

**Comment by dmca-glasgow** [comment]
Hi there, I have the opposite problem!  

I'm looking to turn on the "find in page" functionality but I can't see where to do it.

I'm on MacOS and the latest v1 release of Tauri.

Thanks.

**Comment by samuelint** [comment]
> Hi there, I have the opposite problem!
> 
> I'm looking to turn on the "find in page" functionality but I can't see where to do it.
> 
> I'm on MacOS and the latest v1 release of Tauri.
> 
> Thanks.

**Comment by samuelint** [comment]
@dmca-glasgow 
By anyhow, have you managed to find a way to turn on `find in page`? I'm looking to do the same.
> Hi there, I have the opposite problem!
> 
> I'm looking to turn on the "find in page" functionality but I can't see where to do it.
> 
> I'm on MacOS and the latest v1 release of Tauri.
> 
> Thanks.

**Comment by dmca-glasgow** [comment]
@samuelint No, I didn't find a way.. but I didn't look very hard as it's quite far down my todo list.  If you find a way to enable it, please let me know!

**Comment by mika76** [comment]
I'm also looking how to turn on find in page...
